build.gradle

```diff
@@ -287,6 +290,44 @@ task decompileQuiltflower(type: DecompileTask, dependsOn: [mapNamedJar]) {
 
 	doFirst {
 		file("namedSrc").deleteDir()
+
+		def jdProvider = new MappingsJavadocProvider(insertAutoGeneratedMappings.outputMappings, "named")
+		classJavadocProvider(jdProvider)
+		fieldJavadocProvider(jdProvider)
+		methodJavadocProvider(jdProvider)
+	}
+}
+
+task unpickTargetJar(type: UnpickJarTask) {
+	dependsOn "checkUnpickVersionsMatch", "downloadMinecraftLibraries"
+
+	inputFile.set remapTargetMinecraftJar.outputJar
+	outputFile.set file(Constants.MINECRAFT_VERSION + "-target-named.jar")
+	unpickDefinition.set downloadTargetMappingsJar.targetUnpickDefinitionsFile
+	unpickConstantsJar.set downloadTargetMappingsJar.targetUnpickConstantsFile
+
+	onlyIf {
+		checkUnpickVersionsMatch.match
+	}
+}
+
+task decompileTargetQuiltflower(type: DecompileTask, dependsOn: [unpickTargetJar]) {
+	input.set file(Constants.MINECRAFT_VERSION + "-target-named.jar")
```

Comment:
This should use the output of `unpickTargetJar` directly. I think it can be done with just `input.set unpickTargetJar.outputFile`

Reply:
```suggestion
	input.set unpickTargetJar.outputFile
```